deployment/injector.yml

```diff
@@ -1,6 +1,38 @@
 ---
-- hosts: injector
-  become: yes
-  roles:
-    - user-traffic-reader
-    - injector-configurator
+- hosts:
+    - hyperfoil_controller
+    - hyperfoil_agent
+  tasks:
+  - name: Install Java
+    package:
+      name: java-1.8.0-openjdk-headless
+      state: present
+- hosts:
+    - hyperfoil_controller
+  tasks:
+  - include_role:
+      name: user-traffic-reader
+- hosts:
+    - hyperfoil_controller
+  tasks:
+    - name: Assign path variable
+      set_fact:
+        csv_dest_file_path: "{{ csv_file_path }}/{{ csv_file_name }}"
+    - name: Retrieve the CSV inventory file
+      fetch:
+        src: "{{ csv_file_path }}/{{ csv_file_name }}"
+        dest: "{{ playbook_dir }}/{{ toolkit_csv_file_path }}/"
+        flat: yes
+- hosts:
+    - hyperfoil_controller
+    - hyperfoil_agent
+  tasks:
+    - name: Distribute CSV inventory source file
+      copy:
+        src: "{{ playbook_dir }}/{{ toolkit_csv_file_path }}/{{ csv_file_name }}"
+        dest: "{{ csv_file_path }}/"
+        force: yes
+- hosts: hyperfoil_controller
```

Comment:
why not define like this?

```
- hosts: hyperfoil_controller
  roles:                     
  - injector-setup
```

Seems more clear to me. But both ways are, I guess, equivalent.

Reply:
as a pattern for all the playbooks, not just this example.

Reply:
I'll re-factor these tasks into roles.